src/main/resources/db-scripts/migration.sql

```diff
@@ -1024,3 +1024,10 @@ CREATE INDEX idx_clinical_event_key ON clinical_event_data (`KEY`);
 CREATE INDEX idx_clinical_event_value ON clinical_event_data (`VALUE`);
 CREATE INDEX idx_sample_stable_id ON sample (`STABLE_ID`);
 UPDATE `info` SET `DB_SCHEMA_VERSION`="2.13.1";
+
+##version: 2.13.2
```

Comment:
Note: I have decided to merge my PR separately (see the Slack message I sent you), so this will change to 2.14.1